src/main/java/dev/Roach/GamesFetcher.java

```diff
@@ -0,0 +1,62 @@
+package dev.Roach;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URLEncoder;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.charset.StandardCharsets;
+
+public class GamesFetcher {
+    private final HttpClient client;
+
+    public GamesFetcher(HttpClient client) {
+        this.client = client;
+    }
+
+    public String getGameContainingKeyword(String keyword) {
+        if (keyword == null) {
+            return "Keyword, cannot be a null";
```

Comment:
In the future let's make it throw an exception or return a special object with proper fields to communicate with the user